diagnostic.md

```diff
@@ -57,13 +69,18 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+appahendChild('li').appendChild('body')
```

Comment:
This will not work since it's trying to call a global function that doesn't exist.  One possible way is `$('li').parent().parent().hide();`.